### ISSUE TITLE
Consider supporting pass-through for OTLP metrics

### ISSUE THREAD

**Issue by cartermp** [honeycombio/refinery#515]
For people who use refinery to sample traces _and_ also send along metrics to honeycomb, it can be surprising that they can't "just point their metrics at refinery" and have them pass through. I've not seen anyone expect that the metrics should be sampled in some way, but having to set your endpoint differently for metrics can be surprising.

**Comment by kentquirk** [comment]
Yes -- this is already on the roadmap for sometime "soon" -- meaning we plan to do it but don't have a time commitment yet.

**Comment by neoeinstein** [comment]
Dupe of #449?

**Comment by rasharab** [comment]
It seems like this is still a problem?

SEVERE: Failed to export metrics. Server responded with UNIMPLEMENTED. This usually means that your collector is not configured with an otlp receiver in the "pipelines" section of the configuration. If export is not desired and you are using OpenTelemetry autoconfiguration or the javaagent, disable export by setting OTEL_METRICS_EXPORTER=none. Full error message: unknown service opentelemetry.proto.collector.metrics.v1.MetricsService

**Comment by cartermp** [comment]
@rasharab are you using HTTP or gRPC? For HTTP it should already be supported, but not for gRPC.

**Comment by rasharab** [comment]
Grpc. Thank you for the heads up.

**Comment by TheJokersThief** [comment]
Ah, I missed this when setting up refinery at $current :) It might be worth adding a callout to the docs on `https://docs.honeycomb.io/manage-data-volume/sample/honeycomb-refinery` because it wasn't obvious to me that if we setup with gRPC as our primary method, we would need to send metrics to the HTTP port.
